### DISCUSSION TITLE
How to cancel hint color that jump to reference by lsp_finder

### DISCUSSION THREAD

**Discussion by wiserfz** [nvimdev/lspsaga.nvim#733] [Q&A] [answered]
Hello, when I jump to reference by lsp_finder, it will display hint color.

<img width="665" alt="image" src="https://user-images.githubusercontent.com/38970394/213339706-90743b97-2ed8-4941-a579-9ed612004181.png">

How can I cancel or change it

**Comment by glepnir** [comment]
it will auto disappear

**Reply by wiserfz** [reply]
yes, I know it will auto disappear, but it covers the code and I hope it just cursor jump or change the color and can display the code.

**Comment by glepnir** [comment]
> but it covers the code

what do you mean ? it not disappear ? it actually is a mark layer and blink after jump.  just tell you cursor jump to this line. maybe can add a option to change the blink time slow fast etc.

**Reply by wiserfz** [reply]
> it not disappear ?

Yes, it disappeared, but it covers the code in blink time and I don't want this behavior.

**Reply by glepnir** [reply]
yes I considered that . so add an option to speed the blank time or disabled it

**Reply by wiserfz** [reply]
> yes I considered that . so add an option to speed the blank time or disabled it

Ok, thank you for support~

**Comment by wiserfz** [comment, marked as answer]
For now, `lsp_saga` can't disable the color of link beacon, but sooner or later it will support an option to speed the blink time or disabled it.

**Reply by glepnir** [reply]
not hint. it works link beacon.
